FirebaseAppDistribution/Sources/FIRAppDistribution.m

```diff
@@ -0,0 +1,291 @@
+                                   OIDAuthorizationRequest *request =
+                                       [[OIDAuthorizationRequest alloc]
+                                           initWithConfiguration:configuration
+                                                        clientId:@"319754533822-"
+                                                                 @"osu3v3hcci24umq6diathdm0dipds1fb"
+                                                                 @".apps.googleusercontent.com"
+                                                          scopes:@[
+                                                            OIDScopeOpenID, OIDScopeProfile,
+                                                            OIDScopeTesterAPI
+                                                          ]
+                                                     redirectURL:[NSURL URLWithString:redirectUrl]
+                                                    responseType:OIDResponseTypeCode
+                                            additionalParameters:nil];
+
+                                   // Create an empty window + viewController to host the Safari UI.
+                                   UIWindow *window = [[UIWindow alloc]
+                                       initWithFrame:[UIScreen mainScreen].bounds];
+                                   window.rootViewController = self.safariHostingViewController;
+
+                                   // Place it at the highest level within the stack.
+                                   window.windowLevel = +CGFLOAT_MAX;
+
+                                   // Run it.
+                                   [window makeKeyAndVisible];
+
+                                   // performs authentication request
+                                   [FIRAppDistributionAppDelegatorInterceptor sharedInstance]
+                                       .currentAuthorizationFlow = [OIDAuthState
+                                       authStateByPresentingAuthorizationRequest:request
+                                                        presentingViewController:
+                                                            self.safariHostingViewController
+                                                                        callback:^(
+                                                                            OIDAuthState
+                                                                                *_Nullable authState,
+                                                                            NSError
+                                                                                *_Nullable error) {
+                                                                          self.authState =
+                                                                              authState;
+                                                                          self->_isTesterSignedIn =
```

Comment:
nit: self. isTesterSignedIn looks nicer than _isTesterSignedIn

Reply:
I think we have to set _isTesterSignedIn instead of self.isTesterSignIn because the latter is readonly